packages/mock-block-dock/dev/test-react-block.tsx

```diff
@@ -44,19 +68,36 @@ export const TestReactBlock: BlockComponent<AppProps> = ({ graph }) => {
   return (
     <div ref={blockRef}>
       <h1>
-        Hello {properties.name}! The id of this block is {entityId}
+        <>
+          Hello{" "}
+          {blockEntity?.properties[extractBaseUri(propertyTypes.name.$id)]}! The
+          id of this block is {blockEntity?.metadata.editionId.baseId}
+        </>
       </h1>
       <h2>Block-handled name editing</h2>
       <input
         type="text"
         placeholder="This block's entity's 'name' property"
-        value={properties.name}
+        value={
+          blockEntity?.properties[extractBaseUri(propertyTypes.name.$id)] as
+            | string
+            | undefined
+        }
         onChange={async (event) => {
+          if (!blockEntity) {
+            return;
+          }
           try {
             const { data, errors } = await graphService!.updateEntity({
               data: {
-                entityId,
-                properties: { ...properties, name: event.target.value },
+                entityId: blockEntity.metadata.editionId.baseId,
+                entityTypeId: blockEntity.metadata.entityTypeId,
+                properties: {
+                  ...blockEntity.properties,
+                  [extractBaseUri(propertyTypes.name.$id)]: event.target.value,
+                },
+                leftToRightOrder: blockEntity.linkData?.leftToRightOrder,
+                rightToLeftOrder: blockEntity.linkData?.rightToLeftOrder,
```

Comment:
Until we implement support for JSON patch or something (to be able to express removal of properties), we're requiring the full contents to be passed on update